src/app/announcement/data/events.ts

```diff
@@ -2,9 +2,9 @@ import { AnnouncementEvent } from '../models/announcement-event';
 
 export const ANNOUNCEMENT_EVENTS: AnnouncementEvent[] = [
   {
-    start: new Date('2022-10-18T20:00:00-05:00').getTime(),
-    end: new Date('2022-10-25T20:10:00-05:00').getTime(),
+    start: new Date('2023-01-31T20:00:00-05:00').getTime(),
```

Comment:
Side note: It's already February 1st for me and would 2nd in a few hours, this time is probably in the past?

Reply:
Yes, I figured I'd do a time in the past so it would appear as soon as I deploy in the morning. Thanks for checking! :)